back/config/initializers/filter_parameter_logging.rb

```diff
@@ -8,3 +8,24 @@
 Rails.application.config.filter_parameters += %i[
   password current_password passw secret token _key crypt salt certificate otp ssn import_ideas.pdf import_ideas.xlsx
 ]
+
+# Filter out base64 encoding
+#
+# Filter only exact matches of specified parameter keys. This avoids filtering false positives.
+# E.g. `image` will filter both `image` and `imageUrl`, whereas `^image$` will only filter `image`.
+Rails.application.config.filter_parameters += [
+  /^avatar$/,
+  /^header_bg$/,
+  /^image$/,
+  /^layout_image$/,
+  /^logo$/
+]
+# Custom filter that does 2 things:
+#   1. Removes base64 image encoding from multiloc values, as our WYSIWYG editors enable the addition
+#      of (multiple) images in the string value(s) of mutlilocs, which the FE encodes as base64.
+#   2. Removes base64 encoding from the `file` parameter value, leaving other information intact (e.g. filename).
+Rails.application.config.filter_parameters << lambda do |param, value|
+  if param == 'file' || (CL2_SUPPORTED_LOCALES.include?(param.to_sym) && JSON.generate(value).include?(';base64,'))
```

Comment:
Do we need the `JSON.generate` in there? Not too big of a deal, but knowing this runs on every request, could add a small overhead for large payloads.

Reply:
Removed. This caused one test to fail (where an integer is passed as an example invalid value), so I added a check of `respond_to?(:include?)` to avoid this error.